TombEngine/Objects/TR4/Entity/tr4_baddy.cpp

```diff
@@ -1321,7 +1321,7 @@ namespace TEN::Entities::TR4
 
 		if (pos.has_value())
 		{
-			if ((target.Animation.ActiveState == BADDY_STATE_DODGE || Random::TestProbability(1 / 2.0f)) &&
```

Comment:
Why this was removed?

Reply:
This was removed because he could not stop bullet for some time, instead of instant.